flag_preflight.py

```diff
@@ -0,0 +1,65 @@
+"""
+Naive warp:
+  pico the svg
+  convert everything to cubic
+  split the cubics (we'll just assume too coarse initially)
+  warp start/end and intermediate points directly
+    no fancy tricks with derivatives... my notes said we should have some
+
+Note the lack of:
+  curve simplification
+  intelligent splitting (only when error introduced)
+
+Usage:
+  python naive_warp.py 1f1e6_1f1ec.svg
+  python naive_warp.py noto-emoji/third_party/region-flags/svg/GB-WLS.svg --out_file GB-WLS-waved.svg
+"""
+from absl import app
+from absl import flags
+
+import enum
+from helpers import *
+from lxml import etree
+from picosvg.svg_meta import ntos, splitns, svgns, xlinkns
+import re
+
+DEFAULT_PRECISION = 200
+SHOULD_RENAME = {
+    "store-width": "stroke-width",
+}
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string("out_file", "-", "Output, - means stdout")
+
+
+def _fix_pt(name, attrib):
+    while match := re.search(r"([0-9]+(?:[.][0-9]*)?)\s*pt", attrib[name]):
+        value = attrib[name]
+        new_sz = ntos(round(float(match.group(1)) * 1.25, 2))
+        new_value = value[:match.start()] + new_sz + value[match.end():]
+        attrib[name] = new_value
+
+def main(argv):
+    tree = load_svg(argv, load_to=LoadTo.ETREE)
+
+    el_to_rm = []
+    for el in tree.getiterator("*"):
+        keys = list(el.attrib.keys())
+        for name in keys:
+            # find and eliminate pt's
+            _fix_pt(name, el.attrib)
```

Comment:
should we teach picosvg to do this?

Reply:
Not unless we come up with a more correct solution; this is kind of "good enough for flags"